.circleci/config.yml

```diff
@@ -124,6 +124,7 @@ jobs:
           <<: *restore_cache
       - run: sudo pip install webtest
       - run: sudo pip install coverage
+      - run: sudo pip install configparser
```

Comment:
This undoes the PR referenced by this comment: https://github.com/oppia/oppia/pull/7631#issuecomment-532161587. The backend tests were failing on CircleCi since the configparser module was not being properly installed via pip. I fixed this issue _locally_ at first (macOS) by manually pip installing configparser. That issue (https://github.com/oppia/oppia/issues/7771) is documented here. CircleCi (which gets run on a Linux system) was exhibiting the same behavior as the issue I had on macOS, and re-introducing this line fixed that issue. (Aside: I believe we need to add configparser as a module that is installed by pip, as opposed to something that comes along with the pylint module (https://github.com/oppia/oppia/blob/develop/scripts/install_third_party_libs.py#L211). I can look into that more later, but that is besides the point for CircleCi).